lib/specinfra/command/windows/base/registry_key.rb

```diff
@@ -1,12 +1,12 @@
 class Specinfra::Command::Windows::Base::RegistryKey < Specinfra::Command::Windows::Base
   class << self
     REGISTRY_KEY_TYPES = {
-      :type_string       => 'String',
-      :type_binary       => 'Binary',
-      :type_dword        => 'DWord',
-      :type_qword        => 'QWord',
-      :type_multistring  => 'MultiString',
-      :type_expandstring => 'ExpandString'
+      :type_string            => 'String',
+      :type_binary            => 'Binary',
+      :type_dword             => 'DWord',
+      :type_qword             => 'QWord',
+      :type_multistring       => 'MultiString',
+      :type_expandstring      => 'ExpandString'
     }
```

Comment:
Good point, was going to take a suffix _converted but decided to parse it instead